opm/autodiff/StandardWell_impl.hpp

```diff
@@ -1229,6 +1223,196 @@ namespace Opm
 
 
 
+    template<typename TypeTag>
+    void
+    StandardWell<TypeTag>::
+    updateIPR(const Simulator& ebos_simulator) const
+    {
+        // TODO: not handling solvent related here for now
+
+        // TODO: it only handles the producers for now
+        // the formular for the injectors are not formulated yet
+        if (well_type_ == INJECTOR) {
```

Comment:
There are many TODOs in this PR? I am quite pragmatic when it comes to merging work-in-progress PRs as long as it solves a real and urgent issue. I suggest we discuss some of these TODOs on the Friday hangout.

Reply:
These TODOs are mostly reminders in this functions. They are mostly not causing problems. But can be sources of problems.

Reply:
Yes. There are some other TODOs in this PR. Mostly also reminders. And also, something will need to be addressed in the future for better simulation, while they are not causing problems now.